WalletWasabi.Fluent/Converters/IndentToColorConverter.cs

```diff
@@ -0,0 +1,56 @@
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Avalonia.Controls.Models.TreeDataGrid;
+using Avalonia.Data;
+using Avalonia.Data.Converters;
+using Avalonia.Media;
+
+namespace WalletWasabi.Fluent.Converters;
+
+internal class IndentToColorConverter : IMultiValueConverter
```

Comment:
The way it's done in the **HistoryTable** is not reusable at all because it requires styles bound to a specific data context.
This solution is reusable and flexible, it works for any model. In fact, we should is this approach in the **HistoryTable**, too, instead.

Reply:
I opened #9231 for this.
If you find something that you believe is worth doing to improve the code, please open an issue for it so it won't be lost.